save-orchestrator/src/test/kotlin/com/saveourtool/save/orchestrator/controller/agents/AgentsControllerTest.kt

```diff
@@ -75,7 +75,7 @@ class AgentsControllerTest {
     fun `should build image, query backend and start containers`() {
         val project = Project.stub(null)
         val execution = Execution.stub(project).apply {
-            type = ExecutionType.STANDARD
+            type = TestingType.PRIVATE_TESTS
```

Comment:
just a note: `ExecutionType.GIT` is `TestingType.PRIVATE_TESTS` and `ExecutionType.STANDARD` is `TestingType.PUBLIC_TESTS`, but in tests and stub you changed it in another way

Reply:
Thanks, must've confused them in some places. Will double check